### ISSUE TITLE
Incorrect win percentage displayed on profile

### ISSUE THREAD

**Issue by SkipKayhil** [yasp-dota/yasp#448]
http://yasp.co/players/83748988 - displays 23.08 win percentage with Storm
[filtered to Storm games](http://yasp.co/players/83748988?players.account_id=83748988&players.hero_id=all&with_account_id=&hero_id=17&isRadiant=all&win=all&patch=all&game_mode=all&lobby_type=all&leagueid=all&significant=1) - displays the correct 42.31 win percentage on the big green bar and in the heroes section

After every Storm game I play my winrate with him displayed in the Heroes section has gone down (even when winning, as is the case for my last four or so). It seems to recognize new games being played but not add the wins. When filtered by hero, the correct percentage is displayed at the top bar

**Comment by howardc93** [comment]
Thanks for the report!  A subtle bug where I left out some properties of the match when doing aggregation update at insertion time (an optimization we recently added in order to speed up player index page load times)

https://github.com/yasp-dota/yasp/blob/master/operations.js#L53

**Comment by howardc93** [comment]
To flush the cache you can load another tab and then re-visit.  We may want to flush all the caches since this bug probably affects all of them. . .

**Comment by howardc93** [comment]
Doesn't seem like the manual flush is working, but next deployment should fix

**Comment by SkipKayhil** [comment]
I think I don't understand your instructions because the winrate hasn't changed for me yet


Also, not sure if this is the same bug or not (I'm thinking it may be) but one of my win percentages with a friend is much lower than it should be as well 

http://yasp.co/players/83748988 - win percentage with Jesta Qwky is 4%, should be around 50% I think

What other things on the page are affected? It looks like the last time a hero was played on the profile is wrong as well (Storm says 8 days, should be today)

**Comment by howardc93** [comment]
Same bug.

We might have to deploy new code before the caches can be flushed, this hopefully will happen sometime tomorrow.

I fixed the last played bug a couple days ago, don't think it's deployed yet.

TLDR wait for the next deployment :)

**Comment by howardc93** [comment]
update: @albertcui is busy internshipping and wants to deploy on the weekend.

**Comment by howardc93** [comment]
looks fixed :)
